lms/static/sass/course/_info.scss

```diff
@@ -1,3 +1,8 @@
+
+@import '../edx-pattern-library-shims/base/variables';
```

Comment:
Interesting-- I was using things from this file in my wiki PR, and I didn't do the import. @lizcohen and/or @alisan617 should we always have the import, even though things just "work" without it?